src/main/resources/Confluence/Macros/ConfluenceGallery.xml

```diff
@@ -0,0 +1,723 @@
+      <description>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>description</name>
+        <number>3</number>
+        <prettyName>Macro description</prettyName>
+        <rows>5</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </description>
+      <id>
+        <disabled>0</disabled>
+        <name>id</name>
+        <number>1</number>
+        <prettyName>Macro id</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </id>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+## Get the parameters and put them in lists if necessary
+#set($title = $wikimacro.parameters.title)
+#set($includeString = $wikimacro.parameters.include)
+#set($excludeString = $wikimacro.parameters.exclude)
+#set($pagesString = $wikimacro.parameters.page)
+#set($include = $stringtool.split($includeString, ","))
+#set($exclude = $stringtool.split($excludeString, ","))
+#set($pages = $stringtool.split($pagesString, ","))
+## Display the gallery title if it is set (newlines are needed for the following macro calls to be standalone)
+#if(!$stringtool.isEmpty($title))
+    ==$title==
```

Comment:
All the code should be indented with 2 spaces and not 4. Otherwise put with a tab of 2 spaces and not a tab of 4 spaces. Please fix the code indentation.

Reply:
I had to use four spaces because I copy the code of all developed macros into our internal documentation on GitLab. If the code only has two spaces the GitLab editor breaks indentation when using Tab/Shift+Tab to change the indentation. In this case I will adapt the indentation again for the PR.